### PR TITLE
added sar materials

### PR DESCRIPTION
**Pull request by scottyhq**
- reorganize sidebar table of contents
- add SAR Sentinel1 tutorial

### CODE REVIEW

**Comment by scottyhq** [comment]
@jomey see the preview link above for new sidebar organization and SAR tutorial example

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -29,9 +29,9 @@ jobs:
     - name: Ensure clean notebooks
       run: |
         pip install nb-clean
-        find book -name "*ipynb" | xargs nb-clean check
+        find book -name "*ipynb" | xargs nb-clean check --preserve-cell-metadata
+
```
Comment (jomey) [inline .github/workflows/test.yaml:33]: Nice to see this cleanup step

**Review by jomey — APPROVED** [review]
Looks good to me.

What would you think of adding some box or section, where the core used libraries are highlighted and a link to their docs?

For this one I would think GDAL, Xarray, RioXarray

**Comment by scottyhq** [comment]
> What would you think of adding some box or section, where the core used libraries are highlighted and a link to their docs? 

Yes, good call. I added a line under the learning objectives with links. I'm going to go ahead and merge this, and can follow up with future modifications if need be!

**Pull request merged**